### ISSUE TITLE
Table navigation shortcuts don't function, and the message "Not in a table cell" is announced for some webpages

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#8462]
### Steps to reproduce:
Go to http://www.catalog.update.microsoft.com/Home.aspx , find the first Edit box (which might have already gained focus) and type something like the phrase "Windows 10" in it and press Enter to initiate a search, press Control+Home in the newly opened webpage and press T a couple of times to reach the table which reads as "Table updates."
### Actual behavior:
If, in this table or in fact in all tables on this page, you press Control+ALT+Left, Control+ALT+Right, Control+ALT+Up and Control+ALT+Down to navigate by column or row, NVDA does nothing and just says: "Not in a table cell."
### Expected behavior:
Table navigation keystrokes should work, or if the table doesn't have columns/rows (strangely of course), NVDA should say something like "No table columns" or "No table rows," not something like "Not in a table cell." This table indeed has rows so CTRL+ALT+Up/Down should function as expected.
Even if you go to the page http://www.catalog.update.microsoft.com/Home.aspx itself and locate its tables, you'll see that table navigation shortcuts don't work.
### System configuration:
IE 11 and Chrome 67 64-Bit
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
2018.1, 2018.2.
#### Windows version:
Windows 7 and 10, both Pro and 64-Bit
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
Yes -- as mentioned above

**Comment by amirsol81** [comment]
This can also be duplicated via Gmail's web interface. If, in Gmail's Basic HTML view, you open a message which has two or more replies and you navigate around each reply via H and Shift+H, NVDA identifies each reply heading as a Table. But its table navigation key strokes don't work and we receive the "Not in a table cell" message.

**Comment by amirsol81** [comment]
Just noticed that table navigation key strokes mentioned for the above examples work in Firefox (latest official version), so this is clearly an issue affecting both IE 11 and Chrome 67.

**Comment by leonardder** [comment]
Can you reproduce this in NVDA 2017.4?

**Comment by armick** [comment]
Yes, slightly different behavior. Not getting the expected result anyway.

**Comment by munawarb** [comment]
I get this issue in Firefox so much that when NVDA tells me there's a table, I'm half expecting table navigation keys to not work. So this bug does actually occur in Firefox as well, with Firefox Quantum.

Sometimes the table navigation keys will work for a particular row but as soon as you press CTRL+UP or DOWN, NVDA says "out of table blank."

Unfortunately, most of the places I've seen it happen are things like banking websites so I'm not able to share URLs directly here for example purposes.

**Comment by netblue44** [comment]
I can conferm as well.

**Comment by munawarb** [comment]
I'm able to confirm that table navigation keys also don't work in GMail's standard interface using Firefox Quantum. I tested with JAWS and it works fine. Re: my last comment concerning sensitive websites only, this is the first example that's common that I can share here, otherwise I only have seen this behavior on banking websites.

JAWS has no trouble in any of these cases, even when column counts change and there are nested tables. This leads me to believe that NVDA's table navigation was implemented in a rudimentary fashion and the screen reader is not able to cope with complex structures.

I did notice on another website I was on that table navigation keys worked as long as the cell was plain text. This table was in survey form where the right-hand portion of the row had a radio button in each cell, and the left-hand part of the table was the question. As expected, if I was on one of the radio buttons, NVDA got stuck, but if I was on the cell that showed the question (in plain text,) NVDA moved down to the next row fine.

**Comment by Adriani90** [comment]
@amirsol81, @ghost are you still able to reproduce this on the link in the issue description? I cannot reproduce it in NVDA 2019.1 and Chrome 73.
@netblue44, @munawarb many table issues seems to be solved in Firefox 66.0.2, including the gmail interface.

**Comment by Adriani90** [comment]
Closing as works for me. If you still have this problem, please comment on it and we can reopen the discussion.

**Comment by daric002** [comment]
Opening this one up since I've got this problem again. 
Tried navigating on Chrome 85.0.4183.83 and Firefox 68.6.1esr on the third table on this site: https://webaim.org/articles/nvda/tables.htm

Looks to me like "normal" table. Pressing T to navigate to the table and then directly trying to navigate with ctrl + alt+ arrows just responds "Not in a table cell. Other tables on the same site works well.

I have the same problem on a website I'm building and i'm wondering if my tbale is wrong or if NVDA together is reading it wrong.
